datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/tile/{check_name}/manifest.json

```diff
@@ -2,23 +2,46 @@
   "manifest_version": "2.0.0",
   "app_uuid": "{app_uuid}",
   "app_id": "{integration_id}",
-  "display_on_public_website": false,
+  "display_on_public_website": true,
   "tile": {{
     "overview": "README.md#Overview",
     "configuration": "README.md#Setup",
     "support": "README.md#Support",
     "changelog": "CHANGELOG.md",
-    "description": "",
+    "description": "<FILL IN>",
     "title": "{integration_name}",
-    "media": [],
-    "classifier_tags": [
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "FILL IN Image 1 caption",
+        "image_url": "<FILL IN>"
+      },
+      {
+        "media_type": "image",
+        "caption": "FILL IN Image 2 caption",
+        "image_url": "<FILL IN>"
+      },
+      {
+        "media_type": "image",
+        "caption": "FILL IN Image 3 caption",
+        "image_url": "<FILL IN>"
+      }
+    ],
+    "classifier_tags": [ 
+      "<Add relevant tags from this list: https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags>",
       "Supported OS::Linux",
       "Supported OS::Windows",
-      "Supported OS::macOS"
+      "Supported OS::macOS",
+      "Category::<FILL IN>",
+      "Offering::<FILL IN>",
+      "Queried Data Type::<FILL IN>",
+      "Submitted Data Type::<FILL IN>"
     ]
   }},
   "assets": {{
-    "dashboards": {{}},
+    "dashboards": {{
+      "<FILL IN dashboard short_name>": "assets/dashboards/<FILL IN>.json"
```

Comment:
```suggestion
      "<FILL IN dashboard short_name ex: integration name overview>": "assets/dashboards/<FILL IN>.json"
```

Reply:
Should this also be applied to all other templates, not just tile integration?

Reply:
Yes this should be for the agent integration scaffolding and tile scaffolding!